package.json

```diff
@@ -25,9 +25,9 @@
     "body-parser": "^1.12.0",
     "coffee-script": "^1.9.1",
     "express": "^4.12.1",
-    "logger-sharelatex": "git+https://github.com/sharelatex/logger-sharelatex.git#v1.1.0",
+    "logger-sharelatex": "git+https://github.com/sharelatex/logger-sharelatex.git#v1.5.8",
```

Comment:
logger is now on 1.5.9 https://github.com/sharelatex/logger-sharelatex/blob/master/package.json#L9

Reply:
There isn't a tag for it though!

Reply:
I've tagged logger and updated this branch now.